src/main/java/decodex/storage/Storage.java

```diff
@@ -109,9 +100,15 @@ public void loadRecipesFromDirectory(ModuleManager moduleManager, RecipeManager
      */
     private File[] getAllRecipeFiles() {
         File recipeDirectory = new File(DEFAULT_RECIPE_DIRECTORY);
+
         File[] files = recipeDirectory.listFiles();
+

```

Comment:
Perhaps add a check if `listFiles()` returns null? If so, throw an exception

Reply:
I feel this might be unnecessary since, if the instantiate function throws exception alr if its a file instead of directory, before it actually goes into this function.

Reply:
Understood, did not see the earlier instantiate method.